### PR TITLE
fix(curriculum): remove whitespace when testing user input

### LINKED ISSUE #46735: Registration Form 30 - Too Strict of Test
### Describe the Issue

The test for this challenge is a bit too strict with whitespace.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-forms-by-building-a-registration-form/step-30

### Your code

```html/css
        <label>
          How did you hear about us?
          <select>
            <option></option>
            <option></option>
            <option></option>
            <option></option>
            <option></option>
          </select>
        </label>
```

### Expected behavior

This code should pass

### Screenshots

_No response_

### System

All

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by lulululu12345**
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46735

### CODE REVIEW

File: curriculum/challenges/italian/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ assert.equal(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)'
 Dovresti posizionare il testo prima dell'elemento `select`.
 
 ```js
-assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
+assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.trim().replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
```
Comment (Sboonny) [inline curriculum/challenges/italian/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md:29]: You don't have to change the test for other languages, you should only change English.

File: curriculum/challenges/portuguese/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ assert.equal(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)'
 Você deve colocar o texto antes do elemento `select`.
 
 ```js
-assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
+assert.match(document.querySelector('fieldset:nth-child(3) > label:nth-child(3)')?.innerHTML?.trim().replace(/[\t\n]+/g, ''), /^How did you hear about us\?/);
```
Comment (Sboonny) [inline curriculum/challenges/portuguese/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md:29]: You don't have to change the test for other languages, you should only change English.

**Review by Sboonny — CHANGES_REQUESTED** [review]
I haven't tested the test yet. But this should take care of failing lint test.

Edit:

the test isn't strict

image

File: curriculum/challenges/italian/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md (hunk 1 above)
Comment (lulululu12345) [reply curriculum/challenges/italian/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md]: I have restored this file to its original state.

File: curriculum/challenges/portuguese/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md (hunk 2 above)
Comment (lulululu12345) [reply curriculum/challenges/portuguese/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md]: I have restored this file to its original state.

Comment (Sboonny) [reply curriculum/challenges/portuguese/14-responsive-web-design-22/learn-html-forms-by-building-a-registration-form/60fac56271087806def55b33.md]: Thank you, I will approve after the tests are done 👍

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @lulululu12345, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Review by JordanMooree — APPROVED** [review]
LGTM! Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 🎉

**Pull request merged**